backend/portal/api/portal-api.yml

```diff
@@ -1175,6 +1175,16 @@ components:
       type: array
       items:
         $ref: "#/components/schemas/ontology_element"
+    tissue_element_array:
+      type: array
+      items:
+        allOf:
+          - $ref: "#/components/schemas/ontology_element"
+          - type: object
```

Comment:
once schema 4.0 is in will this need to be combined? We should add some TODO comments to remind us to cleanup after after we finalize 4.0

Reply:
no, this is the intended final form. Only tissue ontology elements have tissue_type, but are otherwise the same as other ontology elements. This is how 'inheritance' is formatted in openAPI spec